src/main/java/nextstep/ladders/domain/Line.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.ladders.domain;
+
+import nextstep.ladders.domain.exceptions.CountOfPersonZeroException;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Line {
+
+    private final PointGenerateStrategy strategy;
+    private final List<Boolean> points;
+
+    public Line(final int countOfPerson, final PointGenerateStrategy strategy) {
+        if (countOfPerson == 0) {
+            throw new CountOfPersonZeroException();
+        }
+        this.strategy = strategy;
+        this.points = dot(countOfPerson);
```

Comment:
```suggestion
        this.points = dot(countOfPerson, strategy);
```
멤버변수로 관리하기보단 이렇게 매개변수로 넘겨주면 좋을 것 같아요!